src/components/pager/pager.stories.mdx

```diff
@@ -184,71 +106,8 @@ using full-screen mode with device or viewport emulation.
   <Story
     name="using custom responsive settings"
     parameters={{ chromatic: { disable: true } }}
```

Comment:
this `parameters` can be removed I think as you've added it to the story in the `.stories.tsx` file. (Indeed I don't think it works to disable Chromatic without doing that.)